starport/cmd/network_request_verify.go

```diff
@@ -0,0 +1,114 @@
+package starportcmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/chaincmd"
+	"github.com/tendermint/starport/starport/pkg/clispinner"
+	"github.com/tendermint/starport/starport/pkg/numbers"
+	"github.com/tendermint/starport/starport/services/network"
+	"github.com/tendermint/starport/starport/services/network/networkchain"
+)
+
+// NewNetworkRequestVerify creates a new request approve
+// command to approve requests for a chain.
+func NewNetworkRequestVerify() *cobra.Command {
+	c := &cobra.Command{
+		Use:     "verify [launch-id] [number<,...>]",
+		Aliases: []string{"accept"},
+		Short:   "Verify the request and simulate the chain genesis from them",
+		RunE:    networkRequestVerifyHandler,
+		Args:    cobra.ExactArgs(2),
+	}
+	c.Flags().AddFlagSet(flagNetworkFrom())
+	c.Flags().AddFlagSet(flagSetHome())
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	return c
+}
+
+func networkRequestVerifyHandler(cmd *cobra.Command, args []string) error {
+	// initialize network common methods
+	nb, err := newNetworkBuilder(cmd)
+	if err != nil {
+		return err
+	}
+	defer nb.Cleanup()
+
+	// parse launch ID
+	launchID, err := network.ParseLaunchID(args[0])
+	if err != nil {
+		return err
+	}
+
+	// get the list of request ids
+	ids, err := numbers.ParseList(args[1])
+	if err != nil {
+		return err
+	}
+
+	n, err := nb.Network()
+	if err != nil {
+		return err
+	}
+
+	// verify the requests
+	if err := verifyRequest(cmd.Context(), nb, n, launchID, ids...); err != nil {
```

Comment:
```suggestion
	// verify the requests
	if err := verifyRequest(cmd.Context(), nb, launchID, ids...); err != nil {
```

We can call the network method inside the `verifyRequest` method

Reply:
Mmh what do you mean? `n` is initialized in `networkRequestVerifyHandler`

Reply:
the network method is not a pointer and doesn't need to be passed to the `verifyRequest` method, we can pass only the nb and call the `n, err := nb.Network()` again inside the `verifyRequest` method

Reply:
but it's only a suggestion, I think it can be more clear in this way

Reply:
I agree on this although it's not a blockker